monorepo/website/src/pages/news/2024-12-09-adding-mpox.mdx

```diff
@@ -0,0 +1,49 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Addition of mpox"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Addition of mpox</b>
+</div>
+
+# Addition of mpox
```

Comment:
```suggestion
# Addition of mpox and other news
```

The post is about more than mpox ;)

Reply:
I just didn't know if that's a very catchy title?